cmd/ops_agent_windows/main_windows.go

```diff
@@ -8,12 +8,42 @@ import (
 
 	"github.com/kardianos/osext"
 	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
 )
 
 const dataDirectory = `Google/Cloud Operations/Ops Agent`
 const serviceName = "Google Cloud Ops Agent"
 
+// Standalone agents to check for before starting.
+var conflictingAgents = map[string]bool{
+	"StackdriverLogging":    true,
+	"StackdriverMonitoring": true,
+}
+
 func main() {
+	// Fail if any standalone agents are running.
```

Comment:
Is it intentional that you are doing this both for installing the agents *and* running the agents? The former will block Googet package installation (and possibly leave the package in a half-installed state?), both will prevent users from intentionally using both with non-conflicting config. What is the desired UX here?

Reply:
Moved this logic to under the Execute() workflow.